ppoprf/src/ppoprf.rs

```diff
@@ -100,6 +263,16 @@ impl ProofDLEQ {
     RistrettoScalar::from_bytes_mod_order_wide(&out)
   }
 
+  // I2OSP2(x): Converts a non-negative integer x into a byte
+  // array of length 2 as described in [RFC8017]. Note that
+  // this function returns a byte array in big-endian byte order.
+  pub fn i2osp2(x: usize) -> [u8; 2] {
+    let mut z = [0u8; 2];
+    let y = &x.to_be_bytes();
+    z.clone_from_slice(&y[y.len() - 2..y.len()]);
```

Comment:
The compiler knows the slice's length, so this can be `copy_from_slice(&y[y.len() - 2..]);` Sadly not `y[-2:]` :)